### DISCUSSION TITLE
How would you like feature requests? And development roadmap plan?

### DISCUSSION THREAD

**Discussion by hailspuds** [microfeed/microfeed#15] [General] [answered]
So firstly, thank you so much for this project. I'm running a test version of the alpha and it's going great. Documentation was good and I got it all up and running pretty quickly.

I was wondering if you are thinking about publishing a development plan/roadmap? 

Or are you happy to take somewhat random feature requests? If so, how do you like them presented? 

- All in one list here? 
- Individually as a discussion under "ideas"? 
- Individually as an issue?

**Comment by wenbinf** [comment, marked as answer]
> I was wondering if you are thinking about publishing a development plan/roadmap?

Yes. I do want to put some kind of roadmap in this github repo - maybe a [Project](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/quickstart-for-projects)?

> Or are you happy to take somewhat random feature requests? If so, how do you like them presented?

Yes. Feature requests are welcome :)

> * All in one list here?
> * Individually as a discussion under "ideas"?
> * Individually as an issue?

I think the [ideas](https://github.com/microfeed/microfeed/discussions/categories/ideas) category is a good place to send/discuss feature requests.

**Reply by hailspuds** [reply]
Ok no worries. 

I've written them up, I'll post them now and see what you think!
